.pre-commit-config.yaml

```diff
@@ -0,0 +1,33 @@
+repos:
+  - repo: local
+    hooks:
+      - id: flake8
+        name: flake8
+        entry: flake8
+        language: python
+        types: [python]
+        log_file: ".artifacts/flake8.pycodestyle.log"
+  - repo: https://github.com/ambv/black
+    rev: 19.3b0
+    hooks:
+      - id: black
+        python_version: python3.7
+  - repo: https://github.com/pre-commit/mirrors-isort
+    rev: v4.3.19
+    hooks:
+      - id: isort
+        python_version: python3.7
```

Comment:
This tool will run with python2.7, but when the next major release of isort happens it will no longer support 2, so this is somewhat pre-emptive.

Reply:
Consider using https://github.com/asottile/reorder_python_imports, it's better than isort and is a pc hook.